src/java.base/share/classes/java/text/Collator.java

```diff
@@ -232,6 +227,18 @@ public static synchronized Collator getInstance() {
 
     /**
      * Gets the Collator for the desired locale.
+     * @apiNote Implementations of {@code Collator} class may produce
```

Comment:
For less l10n savvy readers, I suggest providing a bit more context in the apiNote, such as by explicitly mentioning Swedish and showing what getting the default collator would look like.